README.md

```diff
@@ -558,9 +558,9 @@ More details about AWS IoT Fleet Provisioning by claim can be found here: https:
 
 *Note: If the fleet provisioning feature fails to provision the new key, certificate or thing/device, the device client will abort with fatal error.*
 
-*Note: If a CSR file is not provided, the Device Client will use Claim Certificate and Private key for provisioning the device.*
+*Note: If the CSR file or the Device Private Key is not provided, the Device Client will use Claim certificate and Private key for provisioning the device*
```

Comment:
Shouldn't this say - "if the CSR file is specified without also specifying a private key, the Device Client will..."